dev/index.d.ts

```diff
@@ -35,7 +35,8 @@ export class TestOutputChannel implements OutputChannel {
     public name: string;
     public append(value: string): void;
     public appendLine(value: string): void;
-    public appendLog(value: string, options?: { resourceName?: string, date?: Date }): void
+    public appendLog(value: string, options?: { resourceName?: string, date?: Date }): void;
+    public replace(value): void;
```

Comment:
These changes should be added in a separate PR so we can release them before merging this.